sync/device.ts

```diff
@@ -158,11 +158,11 @@ async function runSshCommand(address: string, command: string) {
 }
 
 export function stopSupervisor(address: string) {
-	return runSshCommand(address, 'systemctl stop resin-supervisor');
+	return runSshCommand(address, 'systemctl stop balena-supervisor');
 }
 
 export function startSupervisor(address: string) {
-	return runSshCommand(address, 'systemctl start resin-supervisor');
```

Comment:
might need to add compatibility for both systemctl service names here, or try both gracefully or something